docs/import_specifications.ADR.md

```diff
@@ -148,14 +148,14 @@ each error type. Currently the error types are:
 
 * `cannot_find_file` if an input file cannot be found
 * `cannot_parse_file` if an input file cannot be parsed
+* `illegal_file_name` if an illegal file name is provided
```

Comment:
What makes the file name illegal?

Reply:
I'm thinking of the case like `?files=file1.csv,,file2.csv,   ,file3.csv`

Where `""` is translated to `"."` by `pathlib` and the whitespace only files, while officially supported by posix, are not something we should support.

Reply:
Got it.